tests/trait_mapping/test_ols.py

```diff
@@ -0,0 +1,49 @@
TBT (narrower term maps to a broader term)'}, {'id': 'Q93.5', 'url': None, 'database': 'ICD-10', 'description': 'Attributed'}, {'id': '612001', 'url': 'http://omim.org/entry/612001', 'database': 'OMIM', 'description': 'E (exact mapping (the terms and the concepts are equivalent))'}], 'iri': 'http://www.orpha.net/ORDO/Orphanet_199318', 'obo_synonym': None, 'is_defining_ontology': True, 'ontology_name': 'ordo', 'is_obsolete': False, 'synonyms': ['Monosomy 15q13.3', 'Del(15)(q13.3)'], 'ontology_iri': 'http://www.orpha.net/ontology/orphanet.owl', 'label': '15q13.3 microdeletion syndrome', 'description': ['15q13.3 microdeletion (microdel15q13.3) syndrome is characterized by a wide spectrum of neurodevelopmental disorders with no or subtle dysmorphic features.'], 'obo_definition_citation': None, 'has_children': False, 'ontology_prefix': 'ORDO', 'annotation': {'hasDbXref': ['ICD-10:Q93.5', 'OMIM:612001', 'UMLS:C2677613'], 'definition_citation': ['orphanet']}, 'short_form': 'Orphanet_199318', 'term_replaced_by': None}, {'is_root': False, 'in_subset': None, '_links': {'self': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318'}, 'graph': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/graph'}, 'jstree': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/jstree'}, 'parents': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/parents'}, 'hierarchicalParents': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/hierarchicalParents'}, 'ancestors': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/ancestors'}, 'hierarchicalAncestors': {'href': 'http://www.ebi.ac.uk/ols/api/ontologies/efo/terms/http%253A%252F%252Fwww.orpha.net%252FORDO%252FOrphanet_199318/hierarchicalAncestors'}}, 'obo_id': 'Orphanet:199318', 'obo_xref': None, 'iri': 'http://www.orpha.net/ORDO/Orphanet_199318', 'obo_synonym': None, 'is_defining_ontology': False, 'ontology_name': 'efo', 'is_obsolete': False, 'synonyms': ['Monosomy 15q13.3', 'Del(15)(q13.3)'], 'ontology_iri': 'http://www.ebi.ac.uk/efo/efo.owl', 'label': '15q13.3 microdeletion syndrome', 'description': ['15q13.3 microdeletion (microdel15q13.3) syndrome is characterized by a wide spectrum of neurodevelopmental disorders with no or subtle dysmorphic features.'], 'obo_definition_citation': None, 'has_children': False, 'ontology_prefix': 'EFO', 'annotation': {'OMIM_definition_citation': ['OMIM:612001'], 'definition_citation': ['orphanet'], 'ICD10_definition_citation': ['ICD10:Q93.5']}, 'short_form': 'Orphanet_199318', 'term_replaced_by': None}]}, '_links': {'self': {'href': 'http://www.ebi.ac.uk/ols/api/terms'}}})
```

Comment:
Could these long strings be copied to one text file each to make the test code a bit more readable?